pennyway-client-iOS/pennyway-client-iOS/Clean/Data/Repository/DefaultChatStompRepository.swift

```diff
@@ -52,7 +52,7 @@ class DefaultChatStompRepository: NSObject, ChatStompRepository {
         let accessToken = KeychainHelper.loadAccessToken() ?? ""
         let deviceName = DeviceInfoManager.getDeviceModelName()
         let deviceId = DeviceInfoManager.getDeviceId()
-        let headers = ["Authorization": "Bearer \(accessToken)", "device-id": "\(deviceId)", "device-name": "\(deviceName)"]
+        let headers = ["Authorization": "Bearer \(accessToken)", "device-id": "\(deviceId)", "device-name": "\(deviceName)", "heart-beat": "0,25000"] // heart-beat 25초
```

Comment:
첫 번째 숫자 0은 클라이언트가 서버에 보낼 heartbeat 간격을 의미하고,
두 번째 숫자 25000d은 서버가 클라이언트에게 보낼 heartbeat 간격을 의미해요!
근데 이 부분은 지금 재서님이랑 어떻게 해야될지 얘기 중이라서 확정되면 수정해놓을게요!

Reply:
네넵
클라이언트는 서버로부터 오는 heartbeat를 받아서, 서버가 여전히 연결된 상태인지 확인해요! 만약 서버로부터 일정 시간 동안 heartbeat를 받지 못하면, 클라이언트는 "서버와 연결이 끊긴 것 같다"고 판단합니다.
지금 코드는 서버가 클라이언트로 보내는 heartbeat로 연결 상태를 판단하는 책임은 클라이언트가 지게 되는 구조입니다!

Reply:
디스코드에 올려놓은 내용 읽어보시면 됩니다.

## heart-beat 헤더의 이해
```
CONNECT // 클라이언트가 서버한테 전달하는 프레임
heart-beat:<cx>,<cy>

CONNECTED // 서버가 클라이언트의 CONNECT 프레임에 응답하는 프레임
heart-beat:<sx>,<sy>
```

### Stomp Spec에서의 정의
- if `<cx>` is 0 (the client cannot send heart-beats) or `<sy>` is 0 (the server does not want to receive heart-beats) then there will be none
- otherwise, there will be heart-beats every MAX(`<cx>`,`<sy>`) milliseconds
- The heart-beat header is `OPTIONAL`. A missing heart-beat header MUST be treated the same way as a “heart-beat:0,0” header, that is: the party cannot send and does not want to receive heart-beats.

### 설명
용어
- `cx`: Client가 서버로 보낼 수 있는 ping/pong 주기를 서버에게 알림
- `cy`: Client가 서버에게서 받을 수 있는 주기
- `sx`, `sy`도 동일한 개념이지만, 서버 관점에서 클라이언트에게 알려줌

### 필요한 이유
- 한 쪽에서 15 seconds 주기로 ping을 전달한다고 설정해도, 다른 한 쪽에서 이를 수신하기 힘든 환경일 수 있음.
- 따라서, 서로 허용 가능한 주기를 교환한 후에 이 중 더 큰 값을 주기로 설정함.
   - 예를 들어, client에서 "15000, 15000"(15초마다 보내고, 받을 수 있어!)라고 보내도, server가 "25000, 25000"이라고 정하면 MAX(`cx`, `sy`), MAX(`sx`, `cy`)로 주기를 25초로 결정함.

### heart-beat 헤더를 정하지 않으면 끊기던 이유?
- Client가 heart-beat를 "0,0"으로 잡으면, "server로 ping을 보내지도, pong을 받지도 않겠다"라고 알리는 셈.
- heart-beat:15000,0: 15초마다 ping은 보낼 건데, 서버로 부터 받진 않을 것
- heart-beat:0,15000: 난 안 보낼 건데, 서버는 15초마다 나한테 보내야 함.
- Client가 heart-beat 헤더를 명시적으로 정하지 않으면, `heart-beat:0,0`과 동일함. (정의에 의해)

Reply:
클라이언트에서 heart-beat를 안 넣거나, 0,0으로 보낸다 하더라도 서버에서 박동 검사는 언제나 25000,25000으로 맞춰지기 때문에 `0,25000`로 맞춰보내는데 이상 없다고 합니다!
자세한 내용은 디코 확인 부탁드립니다~

이렇게 처리해도 될 것 같아요!